tensorrt_llm/_torch/models/modeling_qwen3_next.py

```
@@ -0,0 +1,1254 @@
+        do_finalize: Optional[bool] = True,
+    ) -> torch.Tensor:
+        assert hidden_states.shape[-1] == self.hidden_dim
+        orig_shape = hidden_states.shape
+        hidden_states = hidden_states.view(-1, self.hidden_dim)
+        use_dp_padding = False
+        all_rank_num_tokens = attn_metadata.all_rank_num_tokens
+
+        if not do_finalize:
+            # assert not self.enable_attention_dp
+            ### TODO: support do_finalize == False?
+            assert False
```

Ruff (0.13.1)

Do not `assert False` (`python -O` removes these calls), raise `AssertionError()`

Replace `assert False`

(B011)